staging/src/k8s.io/apiserver/pkg/storage/tests/cacher_test.go

```diff
@@ -901,7 +901,7 @@ func TestWatchBookmarksWithCorrectResourceVersion(t *testing.T) {
 				pod := fmt.Sprintf("foo-%d", i)
 				err := createPod(etcdStorage, makeTestPod(pod))
 				if err != nil {
-					t.Fatalf("failed to create pod %v: %v", pod, err)
+					t.Errorf("failed to create pod %v: %v", pod, err)
```

Comment:
if we cannot send the create and we've failed the test, why don't we simply return from the gofunc immediately?